pnpm-lock.yaml

```diff
@@ -1,9 +1,14 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
you reverted the pnpm lock to an earlier version. Update your pnpm to latest and run install again.

Reply:
Thanks. I'll check that out tomorrow.